src/vm/proftoeeinterfaceimpl.cpp

```diff
@@ -7009,6 +7013,112 @@ HRESULT ProfToEEInterfaceImpl::ResumeRuntime()
     return S_OK;
 }
 
+HRESULT ProfToEEInterfaceImpl::GetEnvironmentVariable(
+    const WCHAR *szName,
+    ULONG       cchValue,
+    ULONG       *pcchValue,
+    __out_ecount_part_opt(cchValue, *pcchValue) WCHAR szValue[])
+{
+    CONTRACTL
+    {
+        // Yay!
+        NOTHROW;
+
+        // Yay!
+        GC_NOTRIGGER;
+
+        // Yay!
+        MODE_ANY;
+
+        // Yay!
+        EE_THREAD_NOT_REQUIRED;
+
+        // Yay!
+        CANNOT_TAKE_LOCK;
+
+
+        PRECONDITION(CheckPointer(szName, NULL_NOT_OK));
+        PRECONDITION(CheckPointer(pcchValue, NULL_OK));
+        PRECONDITION(CheckPointer(szValue, NULL_OK));
+    }
+    CONTRACTL_END;
+
+    PROFILER_TO_CLR_ENTRYPOINT_ASYNC_EX(kP2EEAllowableAfterAttach,
+        (LF_CORPROF,
+        LL_INFO1000,
+        "**PROF: GetEnvironmentVariable\n"));
```

Comment:
or I can change it to `"**PROF: GetEnvironmentVariable \"%S\".\n", szName`. What do you think? @davmason

Reply:
I personally don't think it matters. This is only debug logging, it gets stripped out for release builds. So it's easy to modify the log messages if you need since you are running bits you built anyways.